java-components/cli/src/main/java/io/github/redhatappstudio/jvmbuild/cli/DiagnosticCommand.java

```diff
@@ -0,0 +1,34 @@
+package io.github.redhatappstudio.jvmbuild.cli;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Set;
+
+import com.redhat.hacbs.classfile.tracker.ClassFileTracker;
+import com.redhat.hacbs.classfile.tracker.TrackingData;
+
+import picocli.CommandLine;
+
+@CommandLine.Command(name = "diagnostic", mixinStandardHelpOptions = true, description = "Print diagnostic information")
+public class DiagnosticCommand {
+    @CommandLine.Command(mixinStandardHelpOptions = true, description = "Dump class tracking information")
+    public void classdump(@CommandLine.Parameters() Path fileName) {
```

Comment:
I realise you may want this somewhere else (I wanted to see what was being store in the classes)

Reply:
This seems useful, but 'diagnostic' is not a good name for it. You could have it as a sub command maybe 'diagnostic print-class-tracking-data'

Reply:
Its currently `diagnostic classdump` but I can rename that easily.

Reply:
oops, I misread that, thats ok then.